src/main/java/com/minecolonies/coremod/util/constants/IToolType.java

```diff
@@ -0,0 +1,14 @@
+package com.minecolonies.coremod.util.constants;
+
+public interface IToolType
+{
+    public String getName();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "public" is redundant in this context. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2333)